StyleCop.Analyzers/StyleCop.Analyzers/Lightup/INamedTypeSymbolExtensions.cs

```diff
@@ -11,11 +11,13 @@ internal static class INamedTypeSymbolExtensions
     {
         private static readonly Func<INamedTypeSymbol, INamedTypeSymbol> TupleUnderlyingTypeAccessor;
         private static readonly Func<INamedTypeSymbol, ImmutableArray<IFieldSymbol>> TupleElementsAccessor;
+        private static readonly Func<INamedTypeSymbol, bool> IsSerializableAccessor;
 
         static INamedTypeSymbolExtensions()
         {
             TupleUnderlyingTypeAccessor = LightupHelpers.CreateSyntaxPropertyAccessor<INamedTypeSymbol, INamedTypeSymbol>(typeof(INamedTypeSymbol), nameof(TupleUnderlyingType));
             TupleElementsAccessor = LightupHelpers.CreateSyntaxPropertyAccessor<INamedTypeSymbol, ImmutableArray<IFieldSymbol>>(typeof(INamedTypeSymbol), nameof(TupleElements));
+            IsSerializableAccessor = LightupHelpers.CreateSyntaxPropertyAccessor<INamedTypeSymbol, bool>(typeof(INamedTypeSymbol), nameof(IsSerializable));
```

Comment:
💡 It may make sense to rename `CreateSyntaxPropertyAccessor` to `CreatePropertyAccessor`, especially when it is used for non-syntax types as well.

Reply:
This has more fanout than I first expected, plus the pattern was already being used before this PR, so I left it as-is.